ui-manchette/src/components/Manchette.tsx

```diff
@@ -1,51 +1,217 @@
-import React from 'react';
+import React, { useEffect, useRef } from 'react';
 import OperationalPointList from './OperationalPointList';
-import { OperationalPointType } from '../types';
+import { OperationalPointType, ProjectPathTrainResult, StyledOperationalPointType } from '../types';
 import { ZoomIn, ZoomOut } from '@osrd-project/ui-icons';
-
+import { SpaceTimeChart, PathLayer } from '@osrd-project/ui-spacetimechart';
+import { calcOperationalPointsToDisplay, operationalPointsHeight } from './helpers';
+import {
+  OperationalPoint,
+  PathData,
+  SpaceScale,
+} from '@osrd-project/ui-spacetimechart/dist/lib/types';
+import {
+  BASE_KM_HEIGHT,
+  BASE_OP_HEIGHT,
+  INITIAL_OP_LIST_HEIGHT,
+  INITIAL_SPACE_TIME_CHART_HEIGHT,
```

Comment:
`INITIAL_OP_LIST_HEIGHT` is not used

```suggestion
  BASE_KM_HEIGHT,
  BASE_OP_HEIGHT,
  INITIAL_SPACE_TIME_CHART_HEIGHT,
```

Reply:
yes it is